src/tty/unix.rs

```diff
@@ -166,6 +166,7 @@ fn get_pw_entry(buf: &mut [i8; 1024]) -> Passwd<'_> {
     }
 
     if res.is_null() {
+        // TODO: More informative error? Even just passwd not found?
         die!("pw not found");
```

Comment:
I'd say this should be fine, since it's not a common issue. But changing it to something more verbose would be fine, should be done in this PR if it's desired though, rather than just adding a TODO for later.

Reply:
Yea, sometimes I add TODOs when I'm coding just to make sure I don't forget about them in review. I think I'm actually happy enough with this message though.